A couple minor lint issues in this Go package.

The vast majority of the code in this package follows good, idiomatic Go style, and there are no `go vet` issues.

However, there just a couple minor `golint` issues in the documentation. Resolving these is easy and would make the Go package slightly better. Anyone who'd like to contribute a PR is welcome!

```
$ golint github.com/google/go-github/...
authorizations.go:401:1: comment on exported method AuthorizationsService.CreateImpersonation should be of the form "CreateImpersonation ..."
authorizations.go:423:1: comment on exported method AuthorizationsService.DeleteImpersonation should be of the form "DeleteImpersonation ..."
event_types.go:126:1: comment on exported type IssueActivityEvent should be of the form "IssueActivityEvent ..." (with optional leading article)
github.go:635:1: comment on exported method Client.RateLimit should be of the form "RateLimit ..."
migrations_source_import.go:166:1: comment on exported method MigrationService.ImportProgress should be of the form "ImportProgress ..."
```

There's also one typo in [github.go](https://github.com/google/go-github/blob/6a121d01abf87d70067aa7da6c45c56d6dd41fc7/github/github.go#L411) where "unnecessarily" is misspelled with an extra 'c', and a few places in tests where the value of `err` isn't checked. Fixing these would be great too. See the complete list at https://goreportcard.com/report/github.com/google/go-github.

--- a/github/authorizations.go
+++ b/github/authorizations.go
@@ -398,7 +398,7 @@ func (s *AuthorizationsService) DeleteGrant(id int) (*Response, error) {
 	return s.client.Do(req, nil)
 }
 
-// Create an impersonation OAuth token.
+// CreateImpersonation creates an impersonation OAuth token.
 //
 // This requires admin permissions. With the returned Authorization.Token
 // you can e.g. create or delete a user's public SSH key. NOTE: creating a
@@ -420,7 +420,7 @@ func (s *AuthorizationsService) CreateImpersonation(username string, authReq *Au
 	return a, resp, err
 }
 
-// Delete an impersonation OAuth token.
+// DeleteImpersonation deletes an impersonation OAuth token.
 //
 // NOTE: there can be only one at a time.
 //

--- a/github/event_types.go
+++ b/github/event_types.go
@@ -123,8 +123,9 @@ type GollumEvent struct {
 	Sender *User       `json:"sender,omitempty"`
 }
 
-// DEPRECATED: IssueActivityEvent represents the payload delivered by Issue webhook
-// Use IssuesEvent instead.
+// IssueActivityEvent represents the payload delivered by Issue webhook.
+//
+// Deprecated: Use IssuesEvent instead.
 type IssueActivityEvent struct {
 	Action *string `json:"action,omitempty"`
 	Issue  *Issue  `json:"issue,omitempty"`

--- a/github/github.go
+++ b/github/github.go
@@ -638,6 +638,8 @@ func category(path string) rateLimitCategory {
 	}
 }
 
+// RateLimit returns the core rate limit for the current client.
+//
 // Deprecated: RateLimit is deprecated, use RateLimits instead.
 func (c *Client) RateLimit() (*Rate, *Response, error) {
 	limits, resp, err := c.RateLimits()

--- a/github/migrations_source_import.go
+++ b/github/migrations_source_import.go
@@ -163,7 +163,7 @@ func (s *MigrationService) StartImport(owner, repo string, in *Import) (*Import,
 	return out, resp, err
 }
 
-// QueryImport queries for the status and progress of an ongoing repository import.
+// ImportProgress queries for the status and progress of an ongoing repository import.
 //
 // GitHub API docs: https://developer.github.com/v3/migration/source_imports/#get-import-progress
 func (s *MigrationService) ImportProgress(owner, repo string) (*Import, *Response, error) {

